src/components/Faq/Faq.js

```diff
@@ -100,12 +101,57 @@ const Faq = ({ currentItemData, navData, faqHtml, crumbs, lang }) => {
     };
   }, []);
 
-  React.useEffect(() => {
+  useEffect(() => {
     return () => {
       document.body.removeAttribute(`data-filters`);
     };
   }, []);
 
+  const removeScript = (script) => {
+    if (script) {
+      document.body.removeChild(script);
+    }
+  };
+
+  const generateLdMetadataArr = () => {
```

Comment:
I think this functiuon definition could be moved outside the component scope so that it's not defined on every render if you pass the `questions` as an argument to it.